### PR TITLE
feat(feature flags): Add not_in action and rename contains to in

### PR DESCRIPTION
**Pull request by risenberg-cyberark**
**Issue #, if available:**

## Description of changes:
Schema changes of condition keywords. 

- Add IN action instead of contains
- Add NOT_IN action

**Checklist**

 

* [x] [Meet tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
* [x] Update tests
* [x] Update docs
* [x] PR title follows [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-python/blob/376ec0a2ac0d2a40e0af5717bef42ff84ca0d1b9/.github/semantic.yml#L2)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by risenberg-cyberark** [comment]
@heitorlessa  there ya go

File: aws_lambda_powertools/utilities/feature_flags/feature_flags.py [hunk 1]
```diff
@@ -65,10 +66,12 @@ def _evaluate_conditions(
 
         for condition in conditions:
             context_value = context.get(str(condition.get(schema.CONDITION_KEY)))
-            cond_action = condition.get(schema.CONDITION_ACTION, "")
-            cond_value = condition.get(schema.CONDITION_VALUE)
+            condition_action = condition.get(schema.CONDITION_ACTION, "")
+            condition_value = condition.get(schema.CONDITION_VALUE)
```
Comment (heitorlessa) [inline aws_lambda_powertools/utilities/feature_flags/feature_flags.py:70]:
```suggestion
            cond_action = condition.get(schema.CONDITION_ACTION, "")
            cond_value = condition.get(schema.CONDITION_VALUE)
```

File: aws_lambda_powertools/utilities/feature_flags/schema.py [hunk 2]
```diff
@@ -79,7 +80,7 @@ class SchemaValidator(BaseValidator):
     The value MUST contain the following members:
 
     * **action**: `str`. Operation to perform to match a key and value.
-    The value MUST be either EQUALS, STARTSWITH, ENDSWITH, CONTAINS
+    The value MUST be either EQUALS, STARTSWITH, ENDSWITH, IN
```
Comment (heitorlessa) [inline aws_lambda_powertools/utilities/feature_flags/schema.py:83]: and NOT_IN?

**Review by heitorlessa — CHANGES_REQUESTED** [review]

File: aws_lambda_powertools/utilities/feature_flags/schema.py (hunk 2 above)
Comment (risenberg-cyberark) [reply aws_lambda_powertools/utilities/feature_flags/schema.py:83]: will fix

**Review by heitorlessa — APPROVED** [review]

**Pull request merged**